File: devops-scm-api/src/main/java/com/tencent/devops/scm/api/pojo/RepoListOptions.java
package com.tencent.devops.scm.api.pojo;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class RepoListOptions {
    private Integer page;
    private Integer pageSize;
    private String repoName;
    private String user;
}

File: devops-scm-api/src/main/kotlin/com/tencent/devops/scm/api/pojo/RepoListOptions.kt
package com.tencent.devops.scm.api.pojo

data class RepoListOptions(
    // 页码
    val page: Int? = null,
    // 每页数量
    val pageSize: Int? = null,
    // 仓库名称
    val repoName: String? = null,
    // 用户名称
    val user: String? = null
)
